src/sage/numerical/backends/generic_backend.pyx

```diff
@@ -1723,6 +1723,10 @@ cpdef GenericBackend get_solver(constraint_generation = False, solver = None, ba
         <...sage.numerical.backends.interactivelp_backend.InteractiveLPBackend...>
         sage: p.base_ring()                                                           # optional - sage.rings.number_field
         Rational Field
+        sage: p = get_solver(solver='Matrix', base_ring=GF(3)); p                 # optional - sage.rings.number_field
+        <...sage.numerical.backends.matrix_backend.MatrixBackend...>
```

Comment:
And... actually `GF(3)` is not a good choice for an example; linear programming only makes sense for ordered fields because we have inequalities.

Reply:
I wanted to show that it could be arbitrary, but didn't notice it wouldn't make sense. I replaced with `RR` - is it okay to use `sage.rings.number_field` with it?

Reply:
`RR` does not need a `# optional` annotation.

If you want to test with an exact number field, try `QuadraticField(2)`